test/1.4/events/T_after_const_param.py

```diff
@@ -0,0 +1,13 @@
+# © 2023 Intel Corporation
+# SPDX-License-Identifier: MPL-2.0
+
+import stest
+
+cpu = SIM_create_object("clock", "clock", [["freq_mhz", 1]])
+obj.queue = cpu
+
+obj.trigger = None
+SIM_continue(99999)
+stest.expect_equal(obj.res, [0, 0])
+SIM_continue(2)
+stest.expect_equal(obj.res, [4, 7 << 32 & 4])
```

Comment:
`7 << 32 & 4` evaluates to zero? did you mean to use `|`?

Also, this is a rather specific test, its running time is grossly dominated by startup. Would it make sense to merge it into some other test, maybe `T_after`?

Reply:
Oops, yeah I did mean to use `|`.

> Would it make sense to merge it into some other test, maybe `T_after`?

Not _really_, as `T_after` is meant to test basic functionality, but... one could _force_ it to fit, if you insist.

Reply:
I'd say the scope of T_after is all functional aspects of after statements. It's generally good to keep down the number of tests, for performance reasons.